ops/opset13/add.go

```diff
@@ -6,6 +6,14 @@ import (
 	"gorgonia.org/tensor"
 )
 
+const (
+	// MinAddInput is the minimimum amount of inputs the add operator expects.
+	MinAddInput = 2
+
+	// MaxAddInput is the maximum amount of inputs the add operator accepts.
+	MaxAddInput = 2
+)
```

Comment:
When the number of `min` and `max` inputs is the same, I think it is better to just define `NAddInputs = 2`. And then return this variable for both `GetMinInputs()` and `GetMaxInputs()`

Reply:
Should be checked for all operators.